api/envoy/data/core/v2alpha/health_check_event.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package envoy.data.core.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+option (gogoproto.equal_all) = true;
+
+// [#protodoc-title: Health check logging events]
```

Comment:
If this is logging, doesn't it belong in `envoy.data`?

Reply:
It is in `envoy.data`?

Reply:
My bad, I was focusing on the `core` :)